go.mod

```diff
@@ -0,0 +1,130 @@
+module volcano.sh/volcano-global
+
+go 1.21.10
```

Comment:
keep this with karmada is ok and no need to add toolchain.

Reply:
The Go version of the Karmada version we use is consistent with our current, thanks!